lib/react.js

```diff
@@ -250,6 +250,12 @@ declare module react {
   declare export var ConcurrentMode: ({children: ?React$Node}) => React$Node; // 16.7+
   declare export var StrictMode: ({children: ?React$Node}) => React$Node;
 
+  declare export var Suspense: React$ComponentType<{
+    children?: ?React$Node,
+    fallback: React$Node,
```

Comment:
fallback is optional (for nested Suspense components); missing, undefined, and null should all typecheck I think

Reply:
Ah thanks! I thought it was a required prop.